CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## HEAD
 
+### Added
+
+* Support `Content-Type: application/json` [#143]
```

Comment:
I did not put it under ## 1.8.0 version because I don't know if / when a new version is going to be released :smiley:

Reply:
HEAD is perfect 👍